### DISCUSSION TITLE
十六週和十八週？

### DISCUSSION THREAD

**Discussion by JaYanlloz** [1101-GS4719-C/Python-Community-GS4719-C#2194] [General]
不知道各位對於114年要正式改成16週這件事有什麼看法
其實我覺得目前的16+2的狀態是最舒適的，輕鬆一點的通識課16週結束，比較硬的必修課可以上到18週
這樣不僅可以排開各科期中期末的時間，期末考念必修期末時更是一整個心曠神怡，難道一定要改16嗎，16+2為何不是一個選擇呢？

**Comment by Mminniehuang** [comment]
雖然我也很希望可以排開期中期末，但就我個人經驗來說，其實很多必修都在16週那個禮拜考期末考，結果剩一科或是甚至是只剩一個通識拖到後兩週，導致並沒有因為+2而輕鬆，我是因為住北部比較還好，但南部的同學們真的很哀怨，除了一科考試以外，明明根本沒什麼事了，但來回車票成本太高想回家回不了，直接生生少了一兩個禮拜的假期。

**Comment by wcyeee** [comment]
感覺需要看教授們的想法。 有些教授就算課不輕鬆也是會硬要在16週交完考完（可能教授也想放假嗎），導致我們除了還是要在16週硬拼完，1718週還有其他課所以不能早早回家休息。

但如果像你說的這個狀況，我就覺得很棒！是一個可以讓大家慢慢來好好兼顧通識課的好辦法！

**Comment by Jinyuu17** [comment]
台灣人都喜歡「俗又大碗」，像我就是個堂堂正正台灣人，相同的學費價錢，可以獲得更多的教學內容不是很好嗎。來學校是要學習的，而不是放假，若是在工作階段，有薪水領，那放假絕對是多數人渴望的。

**Comment by 113607514** [comment]
對於改成16週的想法，其實很多人都有類似的感受。16+2的安排確實讓課程負擔更均衡，尤其是對於不同難度的課程來說，這樣可以更好地調整學習節奏。

而且，期中和期末考的時間分配上也比較靈活，可以讓學生有更多的準備時間。改回16週的話，可能會讓某些科目的學習變得緊湊，特別是需要深入理解的必修課，這對學生的壓力來說可能不太友好。

如果能夠在課程安排上保持一些彈性，讓老師和學生都能夠選擇適合的學習模式，或許會是一個更好的解決方案。

**Comment by Tchang323** [comment]
以社團活動來說，16周真的有點趕，尤其是上學期還需要播時間給招生。不過以上課來說，我會比較傾向16周，後面兩周的時間可以自己彈性利用，前面16周課程密集上完、考試完，多兩周真的會讓學生感到疲憊。

**Comment by seastar0687** [comment]
很多課都是16結束，卻要為了一兩堂課，18週才能放假，也很難跟別校的朋友配合，從這個層面來說是更喜歡統一一下，但確實能夠把期末考錯開是對成績更有利，所以終究是看是更想拿好成績還是想好好放松

**Comment by ShuliANN** [comment]
我認同目前16+2學期結構的安排是最理想的。根據我台大的朋友分享，他們的16週學期幾乎每週都被考試和作業塞滿，導致個人休閒時間和培養其他興趣的機會大幅減少。雖然這些活動可以利用學期結束後的假期來進行，但我認為大學時期同樣重要的是建立人際關係和社交網絡。利用在校期間多與同學交流和學習，對未來的發展非常有幫助。

**Comment by lai1008** [comment]
雖然我也很希望可以排開期中期末，但就我個人經驗來說，其實很多必修都在16週那個禮拜考期末考，結果剩一科或是甚至是只剩一個通識拖到後兩週，導致並沒有因為+2而輕鬆。因為我是南部人，除了一科考試以外，明明根本沒什麼事了，但來回車票成本太高想回家回不了，直接生生少了一兩個禮拜的假期。

**Comment by yuan3x2** [comment]
我認為16+2確實多了更多彈性，但相對的也會多一些不確定性，無法提前安排寒假計畫，因此訂好確切時間相對來說帶來了不少方便，而且16+2也有可能遇到課程考試都被安排在同一週的可能，只能說魚與熊掌不可兼得，多數決取勝。

**Comment by christaaah** [comment]
其實我也喜歡16+2，因為通識課的考試會和大課分開，壓力也不會那麼大，除此之外，課程也不會那麼趕，也可以在學期中和其他同學做一些課程外的活動。但是更多人投票選擇16週，也只能遵守規定了。

**Comment by yue083139** [comment]
我個人其實覺得16+2的安排很理想，因為通常都是通識課之類沒什麼進度壓力的課安排在16周考，而需要花時間準備的必修課則在17或18周，其實考到後面沒甚麼課的時候也蠻輕鬆的跟放假沒兩樣，我個人覺得是沒必要一定要跟進頂大改成強制16周啦。

**Comment by forestdew507** [comment]
我比較喜歡 16+2，雖然看到其他校或者都壓在 16 周的系放假會很羨慕哈，但我覺得專業課程的確需要 18 周才能夠把知識以相對完整的程度傳授到學生身上，甚至連 18 周都稍嫌不足，如果砍到 16 周那從課堂解說才比較好理解的東西就更需要課後花時間自己調資料來吸收了~

**Comment by Elaine4488** [comment]
我自己喜歡16+2，期末考錯開比較好準備，也真的減輕蠻多壓力的。學期縮短讓假期變長，可以讓學生自主善用時間，但我認為有些課程太壓縮，實在很痛苦。尤其系上的必修課把原本18週的課程內容變成16週，其實不僅對教授負擔很大，對學生的負擔也蠻大的。

**Comment by Taoyi-Chiang** [comment]
其實16週有一個好處是，可以盡量和外籍生的節慶時間配合。
但是站在我自己的立場，覺得大學16週其實和18週差不了多少吧，反正都教不完要自己回家看🥲

**Comment by Boying12** [comment]
我認為16周更考驗學生的自律性，在有限的時間下更能分出大家的理解與應用能力，而像我這種不擅長讀書的孩子，要得到能看的成績的話，就需要16+2來錯開考試，增加準備時間，以提升成績。不過大四的朋友應該會比較想早早放假。

**Comment by NiCL100** [comment]
期待十六週耶，畢竟真的可以比較早回家，而且朋友大多也十六週，要做什麼時間也比較可以配合。不過每個人都有每個人自己的看法，一律尊重。

**Comment by qiu0109** [comment]
我覺得16週可以早早放假過節感覺很快樂，但是想到如果期末都排在同一週的話會很恐怖，畢竟我是考試前一周才會開始看考古題的人，如果要讓期末錯開的話還是16+2比較好。

**Comment by Mminniehuang** [comment]
原本支持十六週而現在真的面對期末考的我，超級超級反悔。
我一想到差不多一個禮拜之後，就要在五天內直接面對四個必修、一個選修、兩個通識的期末考爆擊，就覺得壓力好大好痛苦😭

**Comment by kzx-creat** [comment]
上學期有系上偏硬的課，因為課堂文本偏多，所以很希望上18周，要不然16周真的超趕看不完。這學期沒有真的硬的課，期末也都是專題報告，就覺得16周OK。

**Comment by poyen0806** [comment]
我覺得16+2的制度確實增加了彈性，但同時也帶來更多的不確定性，讓寒假的計畫難以提早安排。相較之下，訂定固定的時間表會帶來更多便利。此外，16+2的安排也可能導致課程和考試集中在同一週的情況發生。總的來說，這就是魚與熊掌不可兼得的抉擇，最終還是以多數決定為準。

但我第一次覺得16+2好，因為這次是真的16周就結束了

**Comment by lichilun** [comment]
16週制的改變會對學期安排對學生的學習節奏產生影響，這樣的調整有優點有缺點，當視角不同，感受也不一樣。

優點：
1.壓力減少：16週的課程週數稍短，讓學生的假期變長，享受自由時光。
2.彈性安排：學生靈活地安排自學週，增加課程之外的學習空間。
3.提升效率：縮短週數可能促使課程內容更精簡集中，讓學生的學習效率提高。

缺點：
1.課程密度增加：週數減少，課程進度可能會變得更緊湊，學生需要適應更高密度的教學。
2.實驗、實作課影響：像理學院的實驗課、需要長期操作的課程，可能因週數縮短而受到壓縮。
3.期末壓縮：期末考、報告或專案可能會集中在較短的時間內，反而加重壓力。

**Comment by Zheyuan987** [comment]
對課業壓力比較重的科系(理組電資)來說,18周其實根本不夠教完,但是16周的話教授能提早放假

**Comment by Kaz9487** [comment]
對課業比較重的科系來說，我覺得維持16+2是個好選擇。
有些教授想提早放假，16週也能順利完成課程，那就上16週。有些教授考試多，課業重，18週才上的完的話，阿就上18週。
最怕的是要18週才上的完教授去上16週的課程，不管是對學生還是教授都不是一件好事

**Comment by a414414** [comment]
我認為16周對教完所有必修科目，並且將他們學好實在偏困難。
因為如果必修科目具有一定的難度，16周可能沒辦法讓學生完全消化吸收。
而如果是18周，多了兩個禮拜更能讓學生好好掌握學科，也能在考試方面有好的發揮。
現在中央的16+2應該是折中方案，暨能讓有辦法16周教完的學科提早完成，也能多2周的彈性空間讓課程進行更彈性，我覺得也很不錯。

**Comment by yue0903ZZ** [comment]
我比較喜歡 18 週課程，因為繳的學費都一樣，我們應該要學更多東西帶走。
而且 18 週讓課業壓力可以比較減緩，我認為剛剛好。
此外，我認為現在中央 16+2 週的方案讓我覺得不太適應，期末考試時間在不同週，使得我中間不知道要幹嘛，很像一具空殼。
聽說下學年就要真的變 16 週了，希望整體體驗部要太差。

**Comment by yuuu3957** [comment]
我覺得16周或18周都可以，但不太喜歡16+2。
16+2會導致考試很分散，但我個人比較喜歡一次考完的感覺，尤其像系上有些必修有兩次期中，再跟16+2混在一起的話，可能連續一個多月每周都有考試，感覺很煩。
另外有的第16周考試，有的第18周考試，我很常16周的考完，就想放假了，18周的考試讀的就會很不認真或讀得很痛苦。

**Comment by shrshu38** [comment]
感覺需要看教授們的想法。有些教授就算課不輕鬆也 是會硬要在16週交完考完(可能教授也想放假嗎), 導致我們除了還是要在16週硬拼完,1718週還有其他 課所以不能早早回家休息。

但如果像你說的這個狀況,我就覺得很棒!是一個可 以讓大家慢慢來好好兼顧通識課的好辦法!

**Comment by edawdewv** [comment]
我覺得16+2週制度蠻合理，既保留了彈性，又能讓教學安排更有餘裕。通識課在16週內結束，不會佔用太多時間，而必修課如果內容比較多，延伸到18週剛好補足深度，對學生來說反而壓力比較平均。期中期末考也能錯開，不會集中在同一週爆炸，對準備考試非常友善。如果全面改成16週，可能會導致課程壓縮，老師趕進度、學生吸收變少，對教學品質反而有影響。

**Comment by jaychang1027** [comment]
我自己是支持16+2的，改成16周後所有考試都會擠在同一周，真的會扛不住，加上我們系上的必修又比較多，可能會有一天要考好幾科的情況，16+2的話就比較能夠平分這些考試的時間，讓自己有更充裕的時間準備每一科。

**Comment by yurukung** [comment]
其實我蠻喜歡16+2週的，因為現在是16+2週就已經常常會連續幾個禮拜都有考試，感覺一個禮拜一科考試的感覺剛剛好，如果改成16週的話，勢必考試間隔會更少，這樣準備的時間也會被壓縮

**Comment by Pyra-0** [comment]
我自己比較偏好16+2週的配置。一方面換16週的話課程進度會變很趕，知識感覺會吸收不良；另一方面是各科的考試感覺因為週數變少會變更集中，現在16+2的制度下期末已經生不如死了，但好歹各個東西會比較分散，能有更多時間分配進度，不敢想像之後東西更集中地一次來會如何。

**Comment by 77JoyWang77** [comment]
我覺得目前16+2是我比較認同的制度，這樣考試不會全集中在一個禮拜，比較輕鬆的通識課程也能提前結束，讓學生準備期末的考試及報告。

**Comment by brianou583** [comment]
16+2 其實在實務上已經是一種有彈性的安排方式，能兼顧課程強度與學生負擔，也讓教學節奏可以依課程性質調整。強制全面改為16 週，雖然可以讓制度更統一、簡化行政流程，但也可能犧牲了對不同課程需求的彈性處理空間。

從學習者角度來看，原本的16+2安排能讓通識、實作類課程有早點結束的餘裕，同時讓需要深度消化的必修課可以延長安排，這種差異化本身就是一種對學習節奏的尊重。若全面改成16週，可能會導致期末壓力更集中、教學節奏更緊湊，反而對學生與老師都是一種壓縮。

**Comment by alice20040127** [comment]
感覺上我比較喜歡16+2週的彈性方式，有些內容比較多的課就不會因為16週，比之前少了2週可以教學的時間，而上課上的很趕很像要把一堆東西趕在時間內全部塞進學生腦裡的感覺。而且有了彈性的範圍就不會每一科都壓在同一週裡面考試，壓力相對來說就會比較少一些，也會有比較多的時間和心力可以去準備。

**Comment by LINSHIHMIN** [comment]
我覺得現在16+2就很好，有些必修課或是一些授課內容比較豐富的，教授就可以自由運用剩下的兩週，其他可以在16週內完成授課內容的課也不會受到影響。下學期就要開始16週，雖然很開心可以在12月底就放寒假，但其實很擔心自己的學習品質會不會因此大幅下降，而且其實在16+2的情況下，就常常期中考週變成期中考月，所以體感還沒休息多久，就要馬上準備期末考，16週勢必考試跟報告都會更集中，如果為了計算總成績，又把上課時數縮在14週內，那我們真的有辦法在這麼短的時間內，消化掉那麼多課業壓力嗎，不知道，雖然會超級累，但我覺得我應該會因為可以放很多假所以很開心。

**Comment by wuyu06** [comment]
我也支持現在的16+2模式，讓不同課程可以依據需求調整週數，減少期中期末的壓力。如果是16周可能會讓部分課程時間變得緊湊，特別是內容較多的必修課

**Comment by Ting940419** [comment]
其實我覺得第16週跟第18週在整個資訊課程裡，扮演的角色滿不一樣的。第16週通常都是比較偏向收尾的部分，像是期末專題發表、程式實作考，或是整學期內容的總複習。有時候老師會補一些之前沒講完的東西，或者帶我們把學過的東西串起來做最後驗收。這週的好處就是很有「成果驗收」的感覺，會讓人有一種「啊我真的有學到東西」的成就感，但壓力相對也比較大一點，因為剛好碰上其他課也在忙期末，很容易疲勞炸裂。

然後第18週的氛圍就完全不一樣了，整個比較輕鬆，常常是做一些課程回顧、老師發問卷、請大家寫心得，有些老師還會講講產業趨勢、工作方向或分享一些有趣的技術。雖然這週很多人會想翹課（因為也沒什麼硬性作業了），但我覺得如果老師有安排得好，其實是蠻有收穫的。像有一次我們老師在第18週請了一個在業界的學長來分享實務經驗，整場都超實用，大家反而比上課還認真聽。
